packages/amplication-data-service-generator/src/server/static/src/providers/secrets/base/secretsManager.service.base.ts

```diff
@@ -0,0 +1,12 @@
+import { ConfigService } from "@nestjs/config";
+
+export class SecretsManagerServiceBase {
+  constructor(protected readonly configService: ConfigService) {}
+  getSecret<T>(key: string): T {
+    const value = this.configService.get(key);
+    return value;
+  }
+  constants = {
+    JWT_SECRET: "JWT_SECRET",
+  };
```

Comment:
why do we need it?

Reply:
for pre defined must have secrets

Reply:
this is not the responsibility of the secret manager - it should not manage the keys. and, this code is not being used anyway at this PR.